src/containers/AccountConnection.jsx

```diff
@@ -37,10 +37,10 @@ class AccountConnection extends Component {
   connectAccount (auth) {
     const { folderPath } = auth
     const { connector } = this.props
-    let { account } = this.state
+    let { account, successTimeout } = this.state
```

Comment:
@CPatchane @lespacedunmatin Using a `boolean` instead of an `integer` will solve this problem (see my other comments).